### PR TITLE
Clifton Strengths to show on both Fac/Staff and Student Profiles

### LINKED ISSUE #1642: Fac/Staff Clifton Strengths do not show up on profile
I'm not sure when these disappeared, but @ejplatzer and my Clifton Strengths are no longer displaying on the profile page.

### PR DESCRIPTION
**Pull request by charlielee144r**
This PR fixes some logic in fetching a user profile so that Clifton Strengths show on the profiles of both Fac/Staff and Students if it is available (and/or public?). 

Fixes #1642

### CODE REVIEW

File: src/services/user.ts [hunk 1]
```diff
@@ -262,13 +262,16 @@ const getProfileInfo = async (username: string = ''): Promise<Profile | undefine
   if (!profile) return undefined;
 
   const fullName = `${profile?.FirstName} ${profile?.LastName}`;
+  const getCliftonStrengths = await CliftonStrengthsService.getCliftonStrengths(
```
Comment (bennettforkner) [inline src/services/user.ts:265]: I wouldn't call this `getCliftonStrengths`. That name implies that it is a function, not a variable.
```suggestion
  const cliftonStrengths = await CliftonStrengthsService.getCliftonStrengths(
```

**Review by bennettforkner — CHANGES_REQUESTED** [review]
Looking good, just a variable name change that I am requesting.

File: src/services/user.ts (hunk 1 above)
Comment (charlielee144r) [reply src/services/user.ts:265]: That makes sense. Thank you!

**Review by charlielee144r — COMMENTED** [review]
Done!

**Review by bennettforkner — APPROVED** [review]
Looks Good To Me!

**Review by EjPlatzer — APPROVED** [review]

**Pull request merged**